crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -63,10 +63,14 @@ pub fn spawn_admin_interface_tasks<A: InterfaceApi>(
         handle.close_on(stop.map(|_| true)).map(Ok)
     });
 
-    tm.add_conductor_task_ignored(&format!("admin interface, port {}", port), |_stop| {
+    tm.add_conductor_task_ignored(&format!("admin interface, port {}", port), move |_stop| {
         async move {
             let mut active_connections = Vec::new();
             futures::pin_mut!(listener);
+
+            #[cfg(feature = "otel")]
+            let conn_metric = holochain_trace::metric::WebsocketConnectionsMetric::new(port);
+
             // establish a new connection to a client
             while let Some(connection) = listener.next().await {
                 active_connections.retain_mut(|handle: &mut JoinHandle<()>| !handle.is_finished());
```

Comment:
This logic really needs to happen when a connection closes so we track the current state as a metric